lib/_http_server.js

```diff
@@ -1051,6 +1058,10 @@ function emitCloseNT(self) {
   }
 }
 
+function hasBodyHeaders(headers) {
```

Comment:
Can you add jsdoc please?

Reply:
If we resolve the https://github.com/nodejs/node/pull/59778#discussion_r2355517032 comment, I will do it in a follow-up PR so I don't need to re-run the CI, ok?